templates/addons/aurora/mysql-cf.yml

```diff
@@ -0,0 +1,190 @@
+      SecurityGroupEgress:
+        - IpProtocol: '-1'
+          Description: Allow all outbound traffic by default
+          CidrIp: 0.0.0.0/0
+  {{logicalIDSafe .Name}}AuroraSecret:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Name: !Sub '/${App}/${Env}/${Name}/{{logicalIDSafe .Name}}/PostgreSQLSecret'
+      Description: !Sub Aurora main user secret for ${AWS::StackName}
+      GenerateSecretString:
+        SecretStringTemplate: '{"username": "admin"}'
+        GenerateStringKey: "password"
+        ExcludePunctuation: true
+        IncludeSpace: false
+        PasswordLength: 16
+  {{- if .ParameterGroup}}
+  # {{logicalIDSafe .Name}}DBClusterParameterGroup:
+  #   Type: 'AWS::RDS::DBClusterParameterGroup'
+  #   Properties:
+  #     Description: !Ref 'AWS::StackName'
+  #     Family: 'aurora-mysql5.7'
+  #     Parameters:
+  #       character_set_client: 'utf8'
+  {{- else}}
+  {{logicalIDSafe .Name}}DBClusterParameterGroup:
+    Type: 'AWS::RDS::DBClusterParameterGroup'
+    Properties:
+      Description: !Ref 'AWS::StackName'
+      Family: 'aurora-mysql5.7'
+      Parameters:
+        character_set_client: 'utf8'
+  {{- end}}
+  {{logicalIDSafe .Name}}:
+    Type: 'AWS::RDS::DBCluster'
+    Properties:
+      MasterUsername:
+        !Join [ "",  [ {{`'{{resolve:secretsmanager:'`}}, !Ref {{logicalIDSafe .Name}}AuroraSecret, ":SecretString:username}}" ]]
+      MasterUserPassword:
+        !Join [ "",  [ {{`'{{resolve:secretsmanager:'`}}, !Ref {{logicalIDSafe .Name}}AuroraSecret, ":SecretString:password}}" ]]
+      DatabaseName: !Ref {{logicalIDSafe .Name}}InitialDBName
+      Engine: 'aurora-mysql'
+      EngineVersion: '5.7.mysql_aurora.2.07.1'
+      EngineMode: serverless
+      EnableHttpEndpoint: true
+      StorageEncrypted: true
+      KmsKeyId: !Ref {{logicalIDSafe .Name}}AuroraKMSCMK
+      DBClusterParameterGroupName: {{- if .ParameterGroup}} {{.ParameterGroup}} {{- else}} !Ref {{logicalIDSafe .Name}}DBClusterParameterGroup {{- end}}
+      DBSubnetGroupName: !Ref {{logicalIDSafe .Name}}DBSubnetGroup
+      VpcSecurityGroupIds:
+        - !Ref {{logicalIDSafe .Name}}DBClusterSecurityGroup
+      ScalingConfiguration:
+        AutoPause: true
+        MinCapacity: !FindInMap [{{logicalIDSafe .Name}}EnvScalingConfigurationMap, !Ref Env, DBMinCapacity]
+        MaxCapacity: !FindInMap [{{logicalIDSafe .Name}}EnvScalingConfigurationMap, !Ref Env, DBMaxCapacity]
+        SecondsUntilAutoPause: !Ref {{logicalIDSafe .Name}}DBAutoPauseSeconds
+  {{logicalIDSafe .Name}}SecretAuroraClusterAttachment:
+    Type: AWS::SecretsManager::SecretTargetAttachment
+    Properties:
+      SecretId: !Ref {{logicalIDSafe .Name}}AuroraSecret
+      TargetId: !Ref {{logicalIDSafe .Name}}
+      TargetType: AWS::RDS::DBCluster
+  {{logicalIDSafe .Name}}AccessPolicy:
+    Metadata:
+      'aws:copilot:description': 'An IAM ManagedPolicy for your workload to access the {{.Name}} db through Data API'
```

Comment:
I think it's safe for us now to remove this role, since the default task role has access to any secrets that are tagged by copilot and we don't need the `rds-data` permissions